cli/stream_command.go

```diff
@@ -1863,12 +1867,36 @@ func (c *streamCmd) showStreamInfo(info *api.StreamInfo) {
 
 		if s.SubjectTransformDest != "" {
 			filter := ">"
+
 			if s.FilterSubject != "" {
 				filter = s.FilterSubject
 			}
-			cols.AddRowf("Subject Transform", "%s to %s", filter, s.SubjectTransformDest)
+
+			cols.AddRowf("Subject Filter and Transform", "%s to %s", filter, s.SubjectTransformDest)
 		} else {
-			cols.AddRowIfNotEmpty("Subject Filter", s.FilterSubject)
+			if s.SubjectTransformDest == "" && len(s.SubjectTransforms) == 0 {
```

Comment:
nested ifs break my brain completely, can we make this a single case statement?

Reply:
Changed to a switch